docs/reverse-replication/ReverseReplicationUserGuide.md

```diff
@@ -164,6 +164,12 @@ In this case, check if you observe the following:
 
 - ***There is data in Pub/Sub yet not present in source database***
 
+    Check for  logs to ensure that records read being read from PubSub. Logs should have messages like below, which indicate records are read from the buffer.
+    ![DataflowLog](https://services.google.com/fh/files/misc/recordsreadfrompubsub.png)
+
+    Check for logs to see if there are any Connection exception warnings like below. This means that the source database is not reachable and the job keeps retrying to connect, hence nothing gets written to the source database.
```

Comment:
Should we also point to the section saying if not reachable, ensure your Dataflow IPs are allowlisted on the MySQL instance, credentials are correct, MySQL is up etc.

Reply:
done, added a reference to the prerequisite.